tasks.py

```diff
@@ -29,10 +34,10 @@ def requirements(ctx, upgrade=False):
     ]
     if upgrade:
         args.append("--upgrade")
-    ctx.run(
+    ctx_run(
+        ctx,
         f"echo '-e .[dev]' | python -m piptools compile "
         f"{' '.join(args)} - -qo- | sed '/^-e / d' > dev_requirements.txt",
```

Comment:
How does the windows runner work? Does it have bash?

Reply:
Unsure but from the [logs](https://pipelines.actions.githubusercontent.com/mJ2awWhNnBYs06r7hgESwBnBIrvKkXMWwIl1fyScZT5thXu53o/_apis/pipelines/1/runs/2334/signedlogcontent/10?urlExpires=2020-10-15T22%3A31%3A48.9489027Z&urlSigningMethod=HMACV1&urlSignature=zrT5wJ8UIFrCOs5jDVYduv7qOFryWLet%2BwDfZ%2BUBX0s%3D) it seems to use `C:\Program Files\PowerShell\7\pwsh.EXE`